includes/class-wc-siftscience-admin.php

```diff
@@ -506,5 +514,33 @@ private function get_reporting_checkbox_description() {
 			$message = 'Help us improve this plugin by automatically reporting errors and statistics. ';
 			return "$message More info <a target='_blank' href='$url'>here</a>.";
 		}
+
+		/**
+		 * Saves custom fields that aren't covered by WooCommerce's automatic method
+		 */
+		private function save_custom_fields() {
+			if ( ! isset( $_POST['nonce'] ) ) {
+				return;
+			}
+
+			$nonce = sanitize_key( $_POST['nonce'] );
```

Comment:
we bind '_nonce'  do we have 'nonce' 
or is it regarding WOO....

Reply:
I'm still testing that.